src/intl/tr-TR.json

```diff
@@ -70,7 +70,7 @@
       },
       "howApply": {
         "question": "Nasıl başvuru yapmalıyım?",
-        "answer": "Fotoğrafın, kişisel bilgilerin ve e-devletten ulaşabileceğin öğrenci belgesiyle birlikte ön başvuruda bulunabilirsin. Böylece biz bu bilgilerin doğruluğunu onaylayabilir ve sonrasında birlikte destek toplayabileceğin kampanya sayfasını oluşturabiliriz."
+        "answer": "Uçurtma Projesi işlem masraflarını düşürmek için yeni bir blokzinciri ağı olan Avalanche ağına geçiş yapmaktadır. Bu geçiş tamamlanana kadar yeni öğrenci başvurusu yapılmayacaktır. Başvuruların son durumununu Uçurtma Projesi resmi twitter ve instagram hesaplarından takip edebilirsin."
```

Comment:
https://github.com/Ucurtma/ucurtma-app/blob/staging/src/intl/en-US.json#L71

there is also english version of this answer.